test/test_topology_injection.py

```diff
@@ -165,6 +164,18 @@
     {{
         "files": ["*"],
         "modifiers": [
+             {{
+                "links": ["hs2:1 -- sw2:1"],
+                "attributes": {{
+                    "link_attr": "link_value"
+                }}
+           }},
+           {{
+                "ports": ["sw2:1"],
+                "attributes": {{
+                    "port_attr": "port_value"
+                }}
+            }},
```

Comment:
Can we test environment/platform variables here?

Reply:
Actually I think I'll remove this test completely. The injection functionality was extracted to the new pyszn package and is properly tested there.